src/test/fixtures/composes.tsx

```diff
@@ -232,124 +246,298 @@ export const mockComposes = {
+                share_with_accounts: ['serviceAccount:test@email.com'],
+              },
+            },
+          },
+        ],
+      },
+    } as ComposeStatus,
+    '551de6f6-1533-4b46-a69f-7924051f9bc6': {
+      image_status: {
+        status: 'building',
+      },
+      request: {
+        distribution: RHEL_8,
+        image_requests: [
+          {
+            architecture: 'x86_64',
+            image_type: 'azure',
+            upload_request: {
+              type: 'azure',
+              options: {},
+            },
+          },
+        ],
+      },
+    } as ComposeStatus,
+    'b7193673-8dcc-4a5f-ac30-e9f4940d8346': {
+      image_status: {
         status: 'success',
-        type: 'aws.s3',
+        upload_status: {
+          options: {
+            url: 'https://s3.amazonaws.com/b7193673-8dcc-4a5f-ac30-e9f4940d8346-disk.vmdk',
+          },
+          status: 'success',
+          type: 'aws.s3',
+        },
       },
-    },
-  },
-  '7b7d0d51-7106-42ab-98f2-f89872a9d599': {
-    image_status: {
-      status: 'success',
-      upload_status: {
-        options: {
-          url: 'https://s3.amazonaws.com/7b7d0d51-7106-42ab-98f2-f89872a9d599-disk.qcow2',
+      request: {
+        distribution: RHEL_8,
+        image_requests: [
+          {
+            architecture: 'x86_64',
+            image_type: 'vsphere',
+            upload_request: {
+              options: {},
+              type: 'aws.s3',
+            },
+          },
+        ],
+      },
+    } as ComposeStatus,
+    '4873fd0f-1851-4b9f-b4fe-4639fce90794': {
+      image_status: {
+        status: 'success',
+        upload_status: {
+          options: {
+            url: 'https://s3.amazonaws.com/4873fd0f-1851-4b9f-b4fe-4639fce90794-installer.iso',
+          },
+          status: 'success',
+          type: 'aws.s3',
         },
+      },
+      request: {
+        distribution: RHEL_8,
+        image_requests: [
+          {
+            architecture: 'x86_64',
+            image_type: 'image-installer',
+            upload_request: {
+              options: {},
+              type: 'aws.s3',
+            },
+          },
+        ],
+      },
+    } as ComposeStatus,
+    '7b7d0d51-7106-42ab-98f2-f89872a9d599': {
+      image_status: {
         status: 'success',
-        type: 'aws.s3',
+        upload_status: {
+          options: {
+            url: 'https://s3.amazonaws.com/7b7d0d51-7106-42ab-98f2-f89872a9d599-disk.qcow2',
+          },
+          status: 'success',
+          type: 'aws.s3',
+        },
       },
-    },
-  },
+      request: {
+        distribution: RHEL_8,
+        image_requests: [
+          {
+            architecture: 'x86_64',
+            image_type: 'guest-image',
+            upload_request: {
+              options: {},
+              type: 'aws.s3',
+            },
+          },
+        ],
+      },
+    } as ComposeStatus,
+  };
+  return mockComposes[composeId as keyof typeof mockComposes];
```

Comment:
If mockComposes has its types declared at declaration, Typescript will be able to infer this information and we can just write:
```suggestion
return mockComposes[composeId];
```